opm/parser/eclipse/Parser/MessageType.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+  Copyright 2016 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef MESSAGETYPE_H
+#define MESSAGETYPE_H
+
+#include <cstdint>
+#include <string>
+
+namespace Opm {
+
+    enum MessageType {
```

Comment:
Please add an extra namespace for the `MessageType` enum; as the code stands now `Debug` will be a valid symbol in the `Opm` namespace. See e.g. https://github.com/OPM/opm-parser/blob/master/opm/parser/eclipse/EclipseState/Schedule/ScheduleEnums.hpp